apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -374,6 +375,36 @@ export class DetailViewContents extends React.Component {
     });
   };
 
+  handleReopenApplicationClick = () => {
+    this.setState({showReopenApplicationConfirmation: true});
+  };
+
+  handleReopenApplicationCancel = () => {
+    this.setState({showReopenApplicationConfirmation: false});
+  };
+
+  handleReopenApplicationConfirmed = () => {
+    $.ajax({
+      method: 'PUT',
+      url: `/api/v1/pd/applications/${this.props.applicationId}`,
+      contentType: 'application/json',
+      dataType: 'json',
+      data: JSON.stringify({status: 'incomplete'})
```

Comment:
I'm wondering if I should do this in `FormController` as well (setting status here), rather than status being part of `form_data`. Then it'd be consistently set through the status param of the data, rather than sometimes in status and sometimes in form_data. BUT, it'd require a bit of work to refactor at this point.